erpnext_com/www/docs/user/manual/en/using-erpnext/index.md

```diff
@@ -25,7 +25,8 @@ effectively manage your data and collaborate with your co-workers.
 - [Tags](/docs/user/manual/en/using-erpnext/tags)
 
 ## Articles
-1. [Check Link Between Documents](/docs/user/manual/en/using-erpnext/articles/check-link-between-documents)
+1. [Keyboard Shortcuts](/docs/user/manual/en/using-erpnext/articles/keyboard-shortcuts)
```

Comment:
not articles, add it in topics... articles are help articles to teach special scenarios etc. This should be a part of the user manual
also move the images accordingly. No need for a new folder for placing the images, just keep them under... collaboration-tools folder